src/client.cpp

```diff
@@ -118,8 +118,9 @@ void Client::loadMods()
 	// Load builtin
 	scanModIntoMemory(BUILTIN_MOD_NAME, getBuiltinLuaPath());
 
-	// If modding is not enabled, don't load mods, just builtin
-	if (!m_modding_enabled) {
+	// If modding is not enabled, don't load mods, just builtin.
+	// 0xFFFFFFFF is CSM flavour setting for 'disable all'.
+	if (!m_modding_enabled || m_csm_flavour_limits == 0xFFFFFFFF) {
```

Comment:
`|| m_csm_flavour_limits == CSMFlavourLimit::CSM_FL_ALL`
Makes the comment line above superfluous.

Reply:
Ok, duh =)

The function won't work it seems, because it would be true if any flavor is used (?).

Reply:
Good point, that's true. Go ahead with checking the variable directly then.

Reply:
Well i'll do what has been suggested.